test/Gitlab/Tests/Api/TagsTest.php

```diff
@@ -94,10 +94,10 @@ public function shouldCreateRelease($releaseName, $description, $expectedResult)
         $api = $this->getApiMock();
         $api->expects($this->once())
             ->method('post')
-            ->with('projects/1/repository/tags/' . urlencode($releaseName) . '/release', $params)
+            ->with('projects/1/repository/tags/' . str_replace(['/', '.'], ['%2F', '%2E'], $releaseName) . '/release', $params)
```

Comment:
why this change?

Reply:
This was added by https://github.com/m4tthumphrey/php-gitlab-api/pull/368, rebasing so this pull request does not have extra changes.

`AbstractApi::encodePath()` explicitly encodes dots, this lines up with that for testing. https://github.com/m4tthumphrey/php-gitlab-api/blob/52965f4af56bb39dcb5393606b612424eab2a176/lib/Gitlab/Api/AbstractApi.php#L170